project_scrum/data/ir_cron_data.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
```

Comment:
Open question. Should this CRON be in `noupdate` ? This will allow the user to update the frequency without any issue when updating the module

Reply:
You are right, thanks for the input.